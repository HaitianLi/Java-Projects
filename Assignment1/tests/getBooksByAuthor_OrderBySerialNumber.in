add collection books.csv
add collection books2.csv
author Haitian Li
exit