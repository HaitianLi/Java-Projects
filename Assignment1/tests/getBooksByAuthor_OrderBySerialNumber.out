user: 17 books successfully added.

user: 19 books successfully added.

user: should come first 111144 (Haitian Li)
come second with serial 111234 (Haitian Li)
third with serial 111456 (Haitian Li)

user: Ending Library process.
